_includes/specials/policy/en/09-proposals.md

```diff
@@ -0,0 +1,113 @@
+
+Developers have also proposed entirely different ways of adding fees
+to presigned transactions. For example, signing inputs of the
+transaction using `SIGHASH_ANYONECANPAY | SIGHASH_ALL` could allow the
+transaction broadcaster to provide fees by appending additional inputs
+to the transaction without changing the outputs. However, as RBF does
+not have any rule requiring the replacement transaction to have a
+higher “mining score” (i.e. would be selected for a block faster), an
+attacker could pin these types of transactions by creating
+replacements encumbered by low-feerate ancestors. What complicates
+the accurate assessment of the mining score of transactions and
+transaction packages is that the existing ancestor and descendant
+limits are insufficient to bound the computational complexity of this
+calculation. Any transactions it is connected to can influence the
+order in which transactions get picked into a block. A fully-connected
+component, called a _cluster_, can be of any size given current
+ancestor and descendant limits.
+
+A long term solution to address some mempool deficiencies and RBF
+pinning attacks is to [restructure the mempool data structure to track
+clusters][mempool clustering] instead of just ancestor and descendant
+sets. These clusters would be limited in size. A cluster limit would
+restrict the way users can spend unconfirmed UTXOs, but make it
+feasible to quickly linearize the entire mempool using the ancestor
+score-based mining algorithm, build block templates extremely quickly,
+and add a requirement that replacement transactions have a
+higher mining score than the to-be-replaced transaction(s).
+
+Even so, it’s possible that no single set of policies can meet the
+wide range of needs and expectations for transaction relay. For
+example, while batched payments may require a number of unconfirmed
+descendants, this leaves room for pinning package RBF of a shared
+transaction through absolute fees. A proposal for [v3 transaction
+relay policy][topic v3 transaction relay] was developed to allow
+contracting protocols to opt in to a more restrictive set of package
+limits. V3 transactions would only permit packages of size two (one
+parent and one child) and limit the weight of the child. These limits
+would mitigate RBF pinning through absolute fees, and offer some of
+the benefits of cluster mempool without requiring a mempool
+restructure.
+
+[Ephemeral Anchors][topic ephemeral anchors] builds upon the
+properties of v3 transactions and package relay to improve anchor
+outputs further. It removes the dust limit for an anchor output
+belonging to a zero-fee v3 transaction, provided the anchor output is
+spent by a fee-bumping child.  Since the zero-fee transaction must be
+fee-bumped by exactly one child (otherwise a miner would not be
+incentivized to include it in a block), this anchor output is
+“ephemeral” and would not become a part of the UTXO set. The ephemeral
```

Comment:
```suggestion
spent by a fee-bumping child.  Since there would be no incentive to include a zero-fee transaction in a block, it must be
fee-bumped by a child of which there can be only one due to the v3 rules. This prevents the “ephemeral” anchor output
from becoming a part of the UTXO set. The ephemeral
```

Do I recall correctly that the ephemeral anchors proposal would also prescribe that all other outputs on the commitment transactions would be CSV-locked to be unspendable before confirmation?

Reply:
@murchandamus 

I think it's the opposite: "Since this anchor MUST be spent, any spending of other outputs in the
same parent transaction MUST directly double-spend prior spends of the
ephemeral anchor. This causes the 1 block CSV timelock on outputs to be
removed in these situations."  -- https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2022-October/021036.html

Greg's phrasing above was bit unclear to me on first reading, so in my own words: if Alice and Bob have a v3 commitment transaction with a 0-value output (ephemeral anchor), the commitment transaction will only be accepted by relay nodes if it's bundled with a spend of that output.  There's no other way under the v3 rules and the ephemeral anchor rules for the transaction to be accepted for relay.  The only alternative to Alice and Bob would be to broadcast a mutual close that doesn't include a zero-value output.

I think you might be thinking of the existing anchor outputs where 1 OP_CSV is used.

Reply:
Thanks for the correction!